content/docs/cloud/concepts.md

```diff
@@ -1,53 +1,54 @@
+## Using API keys
 
 API keys allow users to access Neon application programming interface.
 
-An API key provides access to any action available to the user. Currently API keys cannot be scoped to the specific Projects. Neon Users can provision multiple API keys. An API key that is no longer needed can be revoked, this action cannot be reverted. Any issued API key is valid forever until it is revoked.
+An API key provides access to any action available to the user. Currently, API keys cannot be scoped to the specific Projects. Neon users can provision multiple API keys. An API key that is no longer needed can be revoked; this action cannot be reverted. Any issued API key is valid forever until it is revoked.
 
-Check out the [API Spec](https://console.neon.tech/api-docs) reference for more information about using the API keys and available API methods.
+Check out the [API Reference](https://console.neon.tech/api-docs) for more information about using the API keys and available API methods.
 
 ## Project
 
-Project is a collection of Postgres databases, Postgres users and other settings on Neon cloud service.
+A Project is a collection of Postgres databases, Postgres users and other settings on Neon cloud service.
 
-Project contains a virtual instance with a Postgres server, also called Compute, as well as the storage used to store the project data. Amount of virtual resources available for the Project is subject to limits defined by the [Free Tier](#free-tier).
+A Project contains a virtual instance with a Postgres server, also called Compute, as well as the storage used to store the Project data. The amount of virtual resources available for the Project is subject to limits defined by the [Free Tier](#free-tier).
 
 Compute is stateless and can be automatically activated and suspended due to user activity.
 
-### Compute Lifecycle
+## Compute Lifecycle
 
 A Compute node in Neon is a stateless Postgres process due to the separation of storage and compute. It has two main states: Active and Idle.
 
-Active means that Postgres is running right now. If there are no active queries for 5 minutes, the activity monitor will gracefully put the corresponding compute node into the idle state to save energy and resources. The activity monitor is conservative, and it treats 'idle in transaction' connections as some activity to do not break an application logic that relies on long-lasting transactions. Yet, it closes all 'idle' connections after 5 minutes of complete inactivity.
+Active means that Postgres is running currently. If there are no active queries for 5 minutes, the activity monitor will gracefully put the corresponding compute node into the Idle state to save energy and resources. The activity monitor is conservative, and it treats 'idle in transaction' connections as some activities do not break an application logic that relies on long-lasting transactions. Yet, it closes all 'idle' connections after 5 minutes of complete inactivity.
```

Comment:
```suggestion
Active means that Postgres is running currently. If there are no active queries for 5 minutes, the activity monitor will gracefully put the corresponding compute node into the Idle state to save energy and resources. The activity monitor is conservative, and it treats 'idle in transaction' connections as some activities to avoid breaking an application logic that relies on long-lasting transactions. Yet, it closes all 'idle' connections after 5 minutes of complete inactivity.
```